src/main/scala/entities/people/Student.scala

```diff
@@ -0,0 +1,10 @@
+package entities.people
+
+import entities.course.Course
+
+class Student extends Person {
+	var studentId: Int = _
```

Comment:
I don't think that a constructor is necessary as it is a data class therefore at the moment no code needs to be executed when it is set up. All the constructor would do is set up the variables. And a lot of the time the empty constructor would be used instead.

Reply:
It's more about me initialising it from the simulator, rather than individually setting all values I could just pass them in, in the constructor like I have for the other generated classes.

Reply:
What do you mean by generated classes because you could just use this notation
```
var f = Foo(){A = "a";B = "b";}
```
Rather than
```
var f = Foo()
f.A = "a"
f.B = "b"
```
And this means that we don't have to write and maintain multiple constructors for the different possible combinations of parameters which need to be passed in